content/doc/pipeline/index.adoc

```diff
@@ -314,7 +316,7 @@ SCM system, and then loading those scripts into Jenkins using the *Pipeline
 Script from SCM* option enabled by the workflow-scm-step plugin, which is one of
 the plugins that the Pipeline plugin depends on and automatically installs.
 
-Loading pipeline scripts from another source via `checkout scm` leverages the
+Loading pipeline scripts from another source using the `checkout scm` command leverages the
```

Comment:
Generally the documentation uses the noun `step` instead of `command` when referring the Pipeline steps.